File: hw/maisincho/u3/HW23Observer/ObserverPy/Application.py
from model.IBM import IBM
from model.Investor import Investor

if __name__ == "__main__":
    s = Investor("Sorros")
    b = Investor("Berkshire")
    ibm = IBM("IBM", 120.00)

    ibm.add_observer(s)
    ibm.add_observer(b)

    ibm.set_price(120.10)
    ibm.set_price(121.00)
    ibm.set_price(120.50)
    ibm.set_price(120.75)
    ibm.set_symbol("IBMTets")

File: hw/suarez/U3/HW22Observer/src/main/java/ec/edu/espe/observer/view/Application.java
package ec.edu.espe.observer.view;
import ec.edu.espe.observer.model.IBM;
import ec.edu.espe.observer.model.Investor;

/**
 *
 * @author BERNARDO
 */
public class Application {

    public static void main(String[] args) {
        Investor s = new Investor("Sorros");
        Investor b = new Investor("Berkshire");

        IBM ibm = new IBM("IBM", 120.00);
        ibm.addObserver(s);
        ibm.addObserver(b);
        ibm.setPrice(100.10);
        ibm.setPrice(100.20);
        ibm.setPrice(100.30);
        ibm.setPrice(100.40);
        ibm.setSymbol("IBMTEST");
    }
}    